source/viewmodel.android.ts

```diff
@@ -23,6 +29,30 @@ export class SelectedAsset extends observable.Observable {
         return Promise.reject(new Error("Not implemented."));
     }
 
+    getImage(options?): Promise<imagesource.ImageSource> {
+        return new Promise<imagesource.ImageSource>((resolve, reject) => {
+            try {
+                resolve(this.decodeUri(this._uri, options));
```

Comment:
These options are our JavaScript options and not the Java BitmapFactory.Options.
Would you update it to something like:
```
var downsampleOptions = new BitmapFactory.Options();
downsampleOptions.inSampleSize = this.getSampleSize(this._uri, options);
resolve(this.decodeUri(this._uri, downsampleOptions);
```